<launch>

<!--<node pkg = "regional_pose_estimate" type= "local_pose_estimate" name="pose_estimate" output="screen">
	<remap from = "/scooter/front_top_scan_filtered" to ="/top_scan" />
</node> -->

<node pkg = "regional_pose_estimate" type= "combine_laser" name="laser_estimate" output="screen"/>

<node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan">
<remap from="cloud_in" to="/iMiev/laser_combined"/>
<remap from="scan" to="/iMiev/topdown_scan"/>
<rosparam>
            
            transform_tolerance: 0.01
            min_height: 0.0
            max_height: 1.0

            angle_min: -3.141569/2 # -M_PI/2
            angle_max: 3.141569/2 # M_PI/2
            angle_increment: 0.0087 # M_PI/360.0
            scan_time: 0.1
            range_min: 0.40
            range_max: 50.0
            use_inf: false

            # Concurrency level, affects number of pointclouds queued for processing and number of threads used
            # 0 : Detect number of cores
            # 1 : Single threaded
            # 2->inf : Parallelism level
            concurrency_level: 1
        </rosparam></node>


</launch>
